app/moves/middleware.js

```diff
@@ -81,6 +94,42 @@ module.exports = {
 
     next()
   },
+  setMoveTypeNavigationContent: (count, label) => {
+    return `<span class="app-filter__count">${count}</span><span class="app-filter__content">${label}</span>`
```

Comment:
This feels a little weird to be building HTML in a middleware. 

The idea of having the components is so that the markup is contained just there. If we updated that, for example the element from a `span` to a `p`, or changed the classname this could easily be missed. 

I think it would be cleaner to have the middleware just build objects that can be passed to the views. Or even better, look at how the [presenters](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/tree/master/common/presenters) are used. It's where we transform API data into a structure the component requires.